neutron_lbaas_dashboard/static/app/core/openstack-service-api/barbican.service.js

```diff
@@ -34,10 +35,12 @@
    * @returns The barbican service API.
    */
 
-  function barbicanAPI(apiService, toastService) {
+  function barbicanAPI($q, apiService, toastService) {
     var service = {
       getCertificates: getCertificates,
-      getSecrets: getSecrets
+      getSecrets: getSecrets,
+      createSecret: createSecret,
```

Comment:
Move createSecret/createCertificate in front of getCertificates?

Reply:
move createCertificate: createCertificate up:

    var service = {
      getCertificates: getCertificates,
      createCertificate: createCertificate,
      getSecrets: getSecrets,
      createSecret: createSecret
    };

certificate ops then secret ops.